sdkproject/Assets/Mapbox/Unity/Editor/PopupSelectionMenu.cs

```diff
@@ -0,0 +1,130 @@
+﻿using UnityEngine;
+using UnityEditor;
+using System.Collections.Generic;
+using System;
+using System.Linq;
+using System.Reflection;
+using Mapbox.Unity.MeshGeneration.Modifiers;
+
+public class PopupSelectionMenu : PopupWindowContent
+{
+
+	private Type _type;
+
+	private Action<UnityEngine.Object> _act;
+
+	private List<Type> _modTypes;
+
+	private SerializedProperty _finalize;
+
+	private int _index = -1;
+
+	private Vector2 scrollPos;
+
+	public override Vector2 GetWindowSize()
+	{
+		return new Vector2(250, 250);
+	}
+
+	public override void OnGUI(Rect rect)
+	{
+		if (_modTypes == null || _modTypes.Count == 0)
+		{
+			var list = AssetDatabase.FindAssets("t:" + _type.Name);
```

Comment:
According to this logic, you are only displaying Modifiers which exist in the project as scriptable objects. 
Fails when no `Modifier` scriptable objects in scene - the dropdown will be empty. This menu should always show all the possible modifiers available in the SDK + any custom modifiers created by user. 

Using reflection to find all modifier classes is one possible solution.